scripts/inner/kubestellar

```diff
@@ -393,9 +393,14 @@ function run_kb_backend() {
     fi
 
     echo "starting kube-bind example-backend in background"
-    kubectl ws root:espw
-    kubectl apply -f "$bindir/../kube-bind/deploy/crd"
-    example-backend --oidc-issuer-client-id=kube-bind --oidc-issuer-client-secret=ZXhhbXBsZS1hcHAtc2VjcmV0 --oidc-issuer-url=http://127.0.0.1:5556/dex --oidc-callback-url=http://127.0.0.1:8080/callback --pretty-name="BigCorp.com" --namespace-prefix="kube-bind-" --cookie-signing-key=bGMHz7SR9XcI9JdDB68VmjQErrjbrAR9JdVqjAOKHzE= --cookie-encryption-key=wadqi4u+w0bqnSrVFtM38Pz2ykYVIeeadhzT34XlC1Y= --consumer-scope="Cluster" &> /dev/null &
+
+    kubectl-kubestellar-get-config-for-space --space-name "espw" --provider-name $PROVIDER_NAME --sm-core-config $SM_CONFIG ${in_cluster} --output "/tmp/espw-conf"
```

Comment:
Can we simply remove `--provider-name $PROVIDER_NAME` here? There should be a correct default for that.